presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an IndexOutOfBoundException during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Add fragment result caching support for ``UNNEST`` queries.
+* Add :func:`poisson_cdf` and :func:`inverse_poisson_cdf` functions.
+* Add memory tracking in ``TableFinishOperator`` which can be enabled by setting the ``table-finish-operator-memory-tracking-enabled`` configuration property to ``true``. Enabling this property can help investigating GC issues on the coordinator, by allowing us to debug whether stats collection uses a lot of memory.
+* Remove spilling strategy ``PER_QUERY_MEMORY_LIMIT`` and instead add configuration property ``experimental.query-limit-spill-enabled`` and session property ``query_limit_spill_enabled``. When ``query_limit_spill_enabled`` is set to ``true`` and the spill strategy is not ``PER_TASK_MEMORY_THRESHOLD``, then we will spill whenever a query uses more than the per-node total memory limit in combined revocable and non-revocable memory.  We will also still spill whenever the memory pool exceeds the spill threshold. This fixes an issue where using the ``PER_QUERY_MEMORY_LIMIT`` spilling strategy could prevent the oom killer from running when the memory pool was full. The issue is still present for the ``PER_TASK_MEMORY_THRESHOLD`` spilling strategy.
```

Comment:
@rschlussel - can we not try to get this within 200-250 chars at max - it is fine to provide people a link to the PR for further clarification

Reply:
how's this? it's  390 characters, but mostly because the property names are long.  I don't think it can be shorter and still convey the relevant information.

```
Remove spilling strategy ``PER_QUERY_MEMORY_LIMIT`` and add configuration property ``experimental.query-limit-spill-enabled`` and session property ``query_limit_spill_enabled``. When set to ``true`` and the spill strategy is not ``PER_TASK_MEMORY_THRESHOLD``, then we will spill whenever a query uses more than the per-node total memory limit in combined revocable and non-revocable memory.
```